apps/etterlatte-saksbehandling-ui/client/src/components/behandling/handlinger/AnnullerBehanding.tsx

```diff
@@ -17,9 +18,11 @@ export default function AnnullerBehandling() {
 
   const behandling = useBehandling()
   const erFoerstegangsbehandling = behandling?.behandlingType === IBehandlingsType.FØRSTEGANGSBEHANDLING
+  const erFoerstegangsbehandlingOgOmstillingsstoenad =
+    behandling?.sakType == SakType.OMSTILLINGSSTOENAD && erFoerstegangsbehandling
 
   const behandles = hentBehandlesFraStatus(behandling?.status ?? IBehandlingStatus.IVERKSATT)
-  if (!behandles) {
+  if (!behandles || erFoerstegangsbehandlingOgOmstillingsstoenad) {
```

Comment:
Så man får lov til å annullere revurdering for oms mao?

Reply:
Usikker på hvorfor man bruker begrepet annullere her, men ja - man får lov til å starte en revurdering - finne ut at den aldri ble fullført og avbryte den slik at behandlingen får status `AVBRUTT`. Dermed låser man ikke muligheten for å opprette nye revurderinger.